Include/cpython/pystate.h

```diff
@@ -206,7 +206,7 @@ struct _ts {
    // A debug build is likely built with low optimization level which implies
    // higher stack memory usage than a release build: use a lower limit.
 #  define Py_C_RECURSION_LIMIT 500
-#elif defined(__s390x__)
+#elif defined(__BIG_ENDIAN__)
```

Comment:
```suggestion
#elif defined(__hppa__) || defined(__powerpc64__) || defined(__s390__) || defined(__sparc__)
```

Reply:
Or we could try to use different values for different arches, given that ppc64 could probably use a higher value. Probably worth double-checking SPARC64, as it has even bigger frames than s390x.